superset/assets/src/dashboard/util/charts/getEffectiveExtraFilters.js

```diff
@@ -54,7 +54,17 @@ const getFilter = (col, op, val) => {
   }
 }
 
-const valueToString = (value) => value ? value.toString() : value;
+const valueToString = (value) => {
+  return value == null ? '' : (typeof value === 'object' && !value.toString ? '[object]' : String(value));
```

Comment:
```suggestion
  return value == null ? '' : (typeof value === 'object' && !value.toString ? '[TO_STRING_NOT_FOUND]' : String(value));
```

Reply:
I've done some error reporting. Plus, I've tested it and the results seem fine.

This function hits the error handling function when we have the below as parameter:
1) null
2) Object.create(null) /* Prototype less object */

Reply:
for `null ` shouldn't this will be taken care by `value == null`